src/components/balance/BalancePlasm.vue

```diff
@@ -1,20 +1,28 @@
 <template>
   <div v-if="isConnected(currentNetworkStatus)">
-    <div class="tw-grid md:tw-auto-cols-max xl:tw-grid-cols-2 tw-gap-4">
-      <Address
-        v-model:isOpen="modalAccount"
-        :address="currentAccount"
-        :address-name="currentAccountName"
-      />
+    <div v-if="isH160 || isSS58">
+      <div class="tw-grid md:tw-auto-cols-max xl:tw-grid-cols-2 tw-gap-4">
+        <WalletH160
+          v-model:isOpen="modalAccount"
+          :address="currentAccount"
+          :address-name="currentAccountName"
+        />
+      </div>
+      <div class="tw-grid md:tw-auto-cols-max xl:tw-grid-cols-2 tw-gap-4 tw-mt-4">
+        <ToggleMetaMask />
+      </div>
     </div>
-    <div
-      v-if="isH160 || isSS58"
-      class="tw-grid md:tw-auto-cols-max xl:tw-grid-cols-2 tw-gap-4 tw-mt-4"
-    >
-      <ToggleMetaMask />
+
+    <div v-else>
+      <div class="tw-grid lg:tw-grid-cols-2 tw-gap-4">
+        <Wallet v-model:isOpen="modalAccount" :wallet-name="currentAccountName" />
+      </div>
+      <div class="tw-grid lg:tw-grid-cols-2 tw-gap-4 tw-mt-8">
+        <Addresses :address="currentAccount" />
+      </div>
     </div>
 
-    <div class="tw-grid tw-grid-cols-1 md:tw-grid-cols-3 tw-gap-y-4 md:tw-gap-4 tw-mt-8">
+    <div class="tw-grid tw-grid-cols-1 md:tw-grid-cols-3 tw-gap-y-8 md:tw-gap-8 tw-mt-8">
       <TotalBalance v-if="accountData" :account-data="accountData" />
       <PlmBalance
```

Comment:
In `PlmBalance.vue` component, add another panel that reads the EVM deposit value. You can read the deposit value for a EVM account with `await api.rpc.eth.getBalance(address)`
![bandicam 2021-11-14 13-23-06-050](https://user-images.githubusercontent.com/40356749/141680866-fde30bc0-907c-439e-8f7d-a9e6f8460a1b.jpg)

You can also withdraw from the EVM deposit to your native SS58 account with `api.tx.evm.withdraw(h160Addr, value)`.
![bandicam 2021-11-14 13-23-39-562](https://user-images.githubusercontent.com/40356749/141680997-6b1c35cf-464e-409e-b8f8-fd7c8cbcf130.jpg)
You must sign with your SS58 account from polkadot extension.